### ISSUE TITLE
$VIMRUNTIME resolution of docs fails for GUI vims

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [mhinz/vim-startify#296]
So, GUI vim implementations have different `$VIMRUNTIME` paths than their embedded vim instances actually use:

MacVim uses …

```vim
escape(fnamemodify(resolve($VIMRUNTIME), ':p'), '\') .'doc' ==#
\ '/Applications/MacVim.app/Contents/Resources/vim/runtime/doc'
```

… whereas VimR (on NeoVim) uses …

```vim
escape(fnamemodify(resolve($VIMRUNTIME), ':p'), '\') .'doc' ==#
\ '/Applications/VimR.app/Contents/Frameworks/SwiftNeoVim.framework/'
\ . 'Versions/A/Resources/vim/runtime/doc'
```

This, unfortunately, breaks the `g:startify_skiplist` resolution of "docs" implemented after #78:

<img width="1019" alt="screen shot 2017-11-29 at 11 44 20" src="https://user-images.githubusercontent.com/200/33390148-adb8e2bc-d4fa-11e7-8e8d-3dc42eb924eb.png">

I can add an additional entry for my own setup; but I'm thinking that maybe “ignore Vim's docs” needs to be a little more robust implementation, with a separate option, instead of using the skip-list?

**Comment by mhinz** [comment]
Hmm, yeah, that features certainly predates embedded Vims. :]

But I wonder how much I can really do about it. The approach could be slightly different for each single GUI and I'm somehow reluctant to do RPC (to get the correct runtime directory) in something that most people use as a start screen.

Maybe simple (not perfect) is better here? Let's be blunt and just ignore all paths that match `runtime/doc/*.txt`? There might be a few false positives, but it should be fine for the majority of use cases.

**Comment by ELLIOTTCABLE** [comment]
@mhinz I think an interactive “add this directory to the ignore-list” feature might be the most user-friendly solution?

Example: mapped to <kbd>i</kbd> by default …

1. hitting <kbd>i</kbd> with the cursor over line `[d]` above adds `/usr/local/.../nvim/runtime/doc/eval.txt` to the ignore-list, prints a message to that effect, and re-renders
2. moving down to the next one and hitting <kbd>i</kbd> again replaces that ignore with one for `/usr/local/.../nvim/runtime/doc/*.txt` … etceteras.

That's substantial implementation-work, though, and probably not worth your effort. Seems perfectly reasonable to WONTFIX this, to me, as the (centralized, user-friendly) solution(s) are all out-of-porportion to the user's own solution … maybe just drop a note in the README? `(=`

**Comment by mhinz** [comment]
The implementation itself would probably be easier than the decision on where to save all that. So, yeah, that's a tad too much. :]

I added a new FAQ to the help, though.

Thanks for bringing this up anyway! ✨
